firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallationsException.java

```diff
@@ -29,6 +29,11 @@
      * Project.
      */
     BAD_CONFIG,
+    /**
+     * The service is currently unavailable. This is a most likely a transient condition and may be
+     * corrected by retrying with a backoff.
```

Comment:
What "backoff" are you referring to?
Exponential backoff?

If this is a public comment, I think I would either omit this addition or frame it like this:
"This is a most likely due to a transient condition and may be corrected by retrying. We recommend exponential backoff when retrying requests."